### DISCUSSION TITLE
Command - Search Bar

### DISCUSSION THREAD

**Discussion by simovicaleksa** [shadcn-ui/ui#2346] [Q&A]
How do I create a search bar similar to combobox but you directly type in the select field. The result should behave something like YouTube search bar or the on the top of the browser where you type urls.

**Comment by jmantonellini** [comment]
I'm trying to create the same component! Playing around the ComboBox didn't work, I've encountered several behavior problems because of the Popover and the Command components changing focus and so.

**Comment by swynnejr** [comment]
I am also working on this today, and having some issues adapting existing components.

**Comment by pablodz** [comment]
hmmm
